provisioning/stream/docker/service/race/Dockerfile

```diff
@@ -1,8 +1,7 @@
 # Build container
 FROM golang:1.23.3-alpine3.20 AS buildContainer
-RUN apk add -U --no-cache bash make curl
-# This is needed for CGO_ENABLED racer
-RUN apk add -U --no-cache gcc g++
+# gcc and g++ are needed for CGO_ENABLED racer
+RUN apk add --update --no-cache bash make curl gcc g++
```

Comment:
I mind a bit that we install some dependencies for the container runtime and some for the compilation using CGO. How can you know  what to omit in case we would like to disable CGO?

Reply:
Hmm? I mean I adjusted the comment to specify that gcc and g++ are for GCO...

Reply:
Ok then, but I think it is same as you would separate it and just remove lines. I'll resolve it

Reply:
It isn't. Splitting the lines creates another layer. It's a good practice to minimize the layers unless you have a good reason not to.